src/deploy/osp_deployer/director.py

```diff
@@ -161,6 +161,19 @@ def upload_update_conf_files(self):
             dest_file = self.home_dir + yaml
             self.upload_file(source_file, dest_file)
 
+            unity_lock = "/unity_container_lock.ini"
```

Comment:
As a developer that has ported code to many operating systems, I'm not a fan of this as the directory separator "/" is hard coded to the file name which makes porting a nightmare.  I'm only going to comment here as an FYI.